### ISSUE TITLE
NodeChef Parse Server discussion

### ISSUE THREAD

**Issue by relatedcode** [relatedcode/ParseAlternatives#255]

**Comment by deeDude** [comment]
I have used Nodechef service and I do not recommend to any one. Their web interface (apps dashboard) is horrendous and I could not get Parse Push notifications to work with Nodechef in any way... Also, and most important of all, their customer support is ZERO! In the end, it is a pretty expensive service, given the quality of their platform...

**Comment by AdityaNayak** [comment]
Cant say about the rest. I just migrated and I cant see my data and unable to find a way to reach their support too.

**Comment by deeDude** [comment]
Yeah, Nodechef's support is simply inexistent... and that is one of the
reasons why i am done with them.

**Comment by chipcerio** [comment]
Glad I bumped into this discussion.  I was planning to use this.  Now I'm looking for alternatives.  Thanks

**Comment by flovilmart** [comment]
You should probably back away from nodechef, not only they run a private fork, but don't contribute back their advancements to the rest of the community.

A greedy company with greedy behaviours, we're removed them from the recommended list of providers.

**Comment by TAProgramming** [comment]
Just to mix it up a little, I have had nothing but excellent service from Nodechef. If I've had a question, they reply within a couple of hours. The speed of the server is great and seems to be reliable.
Also if I've had any issues, they've immediately looked into fixing them with keeping me updated personally.
